modules/core/04-channel/keeper/handshake.go

```diff
@@ -394,43 +370,38 @@ func (k Keeper) ChanOpenConfirm(
 		)
 	}
 
-	// verify multihop proof or standard proof
+	var counterpartyHops []string
```

Comment:
The code below setting the counterparty hops is pretty much repeated in all handlers, so we could rug it under some common function.